pixeltable/share/packager.py

```diff
@@ -345,21 +346,22 @@ def restore(self, bundle_path: Path) -> pxt.Table:
         with tarfile.open(bundle_path, 'r:bz2') as tf:
             tf.extractall(path=self.tmp_dir)
 
-        if self.md is None:
+        if self.bundle_md is None:
             # No metadata supplied; read it from the archive
             with open(self.tmp_dir / 'metadata.json', 'r', encoding='utf8') as fp:
-                self.md = json.load(fp)
+                self.bundle_md = json.load(fp)
```

Comment:
The semantics are a little confusing b/c the `bundle_md` in TablePackager contains dataclasses, but the `bundle_md` in TableRestorer is full JSON. If we do the JSON conversion in the class initializer of TablePackager as noted above, it will also resolve that discrepancy.